org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -56,21 +75,7 @@ class Translator {
                 return this._getEndpointByLocationId(payload.locationId).then((endpoints) => {
                     if (endpoints.length === 0) return undefined;
                     
-                    //Check that the ST endpoint is valid (actually has access to devices)
-                    var endpointPromises = [];
-                    endpoints.forEach((endpointUri) => {
```

Comment:
is this code being removed because it was never necessary for the code path where we have payload already?